$NetBSD: patch-hadrian_bindist_Makefile,v 1.1 2023/01/21 04:49:25 pho Exp $

Don't run `ghc-pkg recache' while installing a binary distribution. See
also "do-install" in Makefile.

--- hadrian/bindist/Makefile.orig	2023-01-12 03:34:04.854847389 +0000
+++ hadrian/bindist/Makefile
@@ -213,7 +213,9 @@ update_package_db: install_bin install_l
 	@echo "Updating the package DB"
 	$(foreach p, $(PKG_CONFS),\
 		$(call patchpackageconf,$(shell echo $(notdir $p) | sed 's/-\([0-9]*[0-9]\.\)*conf//g'),$(shell echo "$p" | sed 's:\0xxx\0:   :g'),$(docdir),$(shell mk/relpath.sh "$(ActualLibsDir)" "$(docdir)"),$(shell echo $(notdir $p) | sed 's/.conf//g')))
+ifneq "$(RECACHE_PACKAGE_DB)" "NO"
 	'$(DESTDIR)$(ActualBinsDir)/$(CrossCompilePrefix)ghc-pkg' --global-package-db "$(DESTDIR)$(ActualLibsDir)/package.conf.d" recache
+endif
 
 install_mingw:
 	@echo "Installing MingGW"
